schema/cassandra/cadence/schema.cql

```diff
@@ -342,6 +342,7 @@ CREATE TYPE checksum (
 );
 
 CREATE TABLE executions (
+  created_time                   timestamp,
```

Comment:
Let's add an updated timestamp too?

Reply:
for this table, last_updated_time is in workflow_execution struct.

Reply:
good catch, that should be fine then

Reply:
there are other type of rows in this table such as tasks. would be nice to define updated timestamp as first class column to be consistent across all entries in this table